include/deal.II/lac/sparse_matrix_ez.h

```diff
@@ -508,11 +508,9 @@ class SparseMatrixEZ : public Subscriptor
             const bool       col_indices_are_sorted = false);
 
   /**
-   * Copy the given matrix to this one.  The operation throws an error if the
-   * sparsity patterns of the two involved matrices do not point to the same
-   * object, since in this case the copy operation is cheaper. Since this
+   * Copy the given matrix to this one. Since this
    * operation is nonetheless not for free, we do not make it available through
-   * <tt>operator =</tt>, since this may lead to unwanted usage, e.g. in copy
+   * <tt>operator =</tt>, in which case may lead to unwanted usage, e.g. in copy
    * arguments to functions, which should really be arguments by reference.
```

Comment:
When talking about other matrix classes
![screenshot - 08122015 - 06 05 45 pm](https://cloud.githubusercontent.com/assets/10951883/9239056/b75489de-411d-11e5-828c-fd31c7a96235.png)

I think I may change all of these in a separated commit